uwblueprint/headsupguys
# Feature/connect module endpoints

## Notion ticket link

[Module API + Page Connections](https://www.notion.so/uwblueprintexecs/Module-API-Page-Connections-d5b56152befe4cd89825bbd2c6bb1a1a)



## Implementation description
* Updated seeding script for the modules schema and added a hard coded Created By field 
* Added the getAll api for modules
* Call the getAll api for the modules page
* Call the delete api for tools and refresh the load after tool has been deleted



## Steps to test
1. Check that you can delete tools from `/admin/dashboard/tools` , pressing delete and confirming on the modal should result in the tool getting deleting and you shouldn't be able to see it on the frontend either. ![image](https://user-images.githubusercontent.com/43117838/129129926-23637fc0-6d71-4199-9008-59fe0006b4e2.png)
2. Should see the modules loaded from the database on `/admin/dashboard/modules` 
![image](https://user-images.githubusercontent.com/43117838/129130034-fbc54382-48e5-4489-a3e6-0159a5d0d69c.png)



## What should reviewers focus on?
* Seeding script/adding the getAll route for the modules
* Loading the modules on the modules page
* Calling the delete api for the tool on the tools page


## Checklist
